### PR TITLE
chore: add tsup and unbuild config to default excludes

### PR DESCRIPTION
**Pull request by dammy001**

### CODE REVIEW

**Review by antfu — APPROVED** [review]

File: packages/vitest/src/defaults.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import type { BenchmarkUserOptions, ResolvedCoverageOptions, UserConfig } from './types'
 
 export const defaultInclude = ['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
-export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**', '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress}.config.*']
+export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**', '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*']
```
Comment (sheremet-va) [inline packages/vitest/src/defaults.ts:4]: I think we also need to update documentation?

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: packages/vitest/src/defaults.ts (hunk 1 above)
Comment (dammy001) [reply packages/vitest/src/defaults.ts:4]: @dammy001 docs updated

Comment (sheremet-va) [reply packages/vitest/src/defaults.ts:4]: There are also changes to coverage excludes, which are not documented

Comment (dammy001) [reply packages/vitest/src/defaults.ts:4]: @sheremet-va updated now.

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**